pkgs/development/python-modules/reproject/default.nix

```
@@ -24,6 +25,12 @@ buildPythonPackage rec {
 
   nativeBuildInputs = [ astropy-helpers cython ];
 
+  # Fix tests
+  patches = [ (fetchpatch {
+    url = "https://github.com/astropy/reproject/pull/218.patch";
```

this url isn't stable, please use commit patch
```suggestion
    url = "https://github.com/astropy/reproject/pull/218/commits/4661e075137424813ed77f1ebcbc251fee1b8467.patch";
```